recipes/install.rb

```diff
@@ -130,7 +130,7 @@
   package deb_pkgs do
     options node['mongodb']['packager_options']
     action :install
-    version package_version
+    version [ package_version, package_version, package_version, package_version ]
```

Comment:
what on earth is happening here?

Reply:
It fixes the version of every package.
Now it works only for the first one.
All described in issue 
https://github.com/sous-chefs/mongodb/issues/220

Reply:
Check documentation https://docs.chef.io/resource_package.html `chapter Multiple Packages`

Reply:
ugh, and that's why I never use that "feature". It would be one thing if there was a `deb_pkg_versions` variable that actually matched up.

How do you feel about:
```ruby
  deb_pkg_suffixes = %w(
    server
    shell
    tools
    mongos
  )
  deb_pkg_suffixes.each do |suffix|
    package "#{node['mongodb']['package_name']}-#{suffix}" do
      options node['mongodb']['packager_options']
      action :install
      version package_version
      not_if { node['mongodb']['install_method'] == 'none' }
    end
  end
```

Is that worse? Or just differently terrible?

Reply:
I think it's better. I will commit changes now, thanks.

Reply:
Thank you @josephholsten for pointing me to a better solution!
I've committed changes and tested them locally.

P.S. I used this strange solution before just because wanted to fix my problem and didn't think about little refactoring.